helm_deploy/hmpps-assess-risks-and-needs/values.yaml

```diff
@@ -32,6 +32,8 @@ generic-service:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
       API_CLIENT_ID: "API_CLIENT_ID"
       API_CLIENT_SECRET: "API_CLIENT_SECRET"
+      ORDS_CLIENT_ID: "ORDS_CLIENT_ID"
```

Comment:
The secret `hmpps-assess-risks-and-needs` is managed by the auth team and can be automatically cycled if required so we would lose any keys you're adding here if that were to happen. I would create a new secret here e.g. `hmpps-assess-risks-and-needs-ords` - if you need help doing so give me a shout 👍

Reply:
Thanks for the heads up. Will update this.